app/src/main/res/drawable/thumb_hover.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="55.25dp"
+    android:viewportWidth="200"
+    android:viewportHeight="55.25">
+  <path
+      android:pathData="m23.62,15.62c-5.21,0 -9.48,1.48 -11.72,4.65L11.90,0L0,0L0,54.32L12.09,54.32L12.09,34.04c0,-4.83 2.79,-7.44 7.62,-7.44 4.27,0 6.32,2.41 6.32,6.7L26.04,54.32L37.95,54.32L37.95,45.95 37.95,32.55C37.95,21.76 32,15.62 23.62,15.62ZM62.69,15.62c-11.72,0 -20.83,8.18 -20.83,19.90 0,11.53 9.11,19.71 20.83,19.71 11.72,0 20.83,-8.18 20.83,-19.71C83.53,24 74.41,15.62 62.69,15.62ZM62.69,45.02c-5.39,0 -8.55,-3.90 -8.55,-9.48 0,-5.76 3.35,-9.48 8.55,-9.48 5.39,0 8.55,3.90 8.55,9.48 0,5.58 -3.16,9.48 -8.55,9.48zM130.60,16.55h-10.04c-4.09,0 -9.11,2.41 -11.53,8.37l-1.48,2.79c-1.67,3.90 -2.97,6.51 -2.97,6.51l-0.37,1.67 -0.55,-1.67c0,0 -1.30,-2.78 -2.97,-6.51L99.34,24.74C96.93,18.79 91.72,16.37 87.81,16.37L77.76,16.37c0,0 4.09,1.11 8.37,8.37 4.27,7.06 15.99,30.32 15.99,30.32h4.28c0,0 11.72,-23.25 16,-30.32 4.09,-7.06 8.18,-8.18 8.18,-8.18zM146.23,15.63c-11.71,0 -20.83,8.18 -20.83,19.71 0.00,11.72 8.74,19.90 22.32,19.90 8,0 12.46,-2.41 16.18,-5.58l-6.88,-7.44a16.79,16.79 0,0 1,-9.48 2.97c-5.39,0 -8.55,-2.23 -9.86,-6.14h28.83C167.07,25.86 159.81,15.62 146.23,15.62ZM137.67,31.81c0.92,-4.46 4.09,-6.69 8.92,-6.69 4.65,0 7.44,2.41 8,6.69zM193.67,15.63c-3.90,0 -8.37,1.30 -10.60,5.39L183.06,16.37L170.97,16.37v37.76h12.09L183.06,37.39c0,-8 2.97,-10.60 8.37,-10.60a10.08,10.08 0,0 1,4.65 1.11L200,16.93c-1.48,-0.55 -3.72,-1.30 -6.32,-1.30z"
+      android:strokeWidth="1.72"
+      android:fillColor="#ffffff"/>
+</vector>

```

Comment:
This logo is badly visible on the light background, we should probably better use this one: https://upload.wikimedia.org/wikipedia/commons/4/44/Hover_logo.svg

Reply:
will fix in an instant

Reply:
I broke the one path used for the whole word into several paths for each single letter. Android Studio complains if a path is longer than 800 characters and now its less